packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/SelectAllColumnsCheckbox/SelectAllColumnsCheckbox.component.js

```diff
@@ -24,7 +24,7 @@ const SelectAllColumnsCheckbox = ({ id, onChange, value, t }) => (
 			dataFeature="column-chooser.select.all"
 			description={getDescription(value, t)}
 			label={t('CHECKBOX_VISIBILITY_LABEL', {
```

Comment:
I know it's a little out of scope, but can you change the key to add context for translation team please ?
```suggestion
			label={t('TC_COLUMN_CHOOSER_SELECT_ALL', {
```